### PR TITLE
GUACAMOLE-292: Document new user profile columns of guacamole_user.

### PR DESCRIPTION
**Pull request by mike-jumper**

### CODE REVIEW

**Comment by mike-jumper** [comment]
**NOTE:** Intended for `staging/0.9.13-incubating`.

File: src/chapters/jdbc-auth.xml [hunk 1]
```diff
@@ -805,6 +805,48 @@ postgresql-user-required: true</programlisting>
                             zones.</para>
                     </listitem>
                 </varlistentry>
+                <varlistentry>
+                    <term><property>full_name</property></term>
+                    <listitem>
+                        <para>The user's full name. Unlike the username, this name need not be
```
Comment (jmuehlner) [inline src/chapters/jdbc-auth.xml]: Not that I think it's super needed, but why does this paragraph not have a thing about setting the column to NULL like the others?

Comment (mike-jumper) [inline src/chapters/jdbc-auth.xml]: I'll add a thing.

Comment (mike-jumper) [inline src/chapters/jdbc-auth.xml]: There - hopefully that adds some clarity.

Comment (jmuehlner) [inline src/chapters/jdbc-auth.xml]: #CONSISTENCY

**Pull request merged**